src/FSharpPlus/Validations.fs

```diff
@@ -77,12 +77,14 @@ module Validation=
         | Success a -> f a
 
     /// orElse v a returns 'a when v is Failure, and the a in Success a.
+    [<System.Obsolete("This function will not be supported in future versions.")>]
     let orElse v (a: 'a) = 
         match v with
         | Failure _ -> a
         | Success x -> x
 
     /// Return the 'a or run the given function over the 'e.
+    [<System.Obsolete("This function will not be supported in future versions.")>]
```

Comment:
Great, but for these functions the reason is not the same. We can put here "Use Validation.defaultWith instead."

Reply:
We should add the `defaultWith` and `defaultValue` functions as well